src/components/panel/panel.js

```diff
@@ -923,9 +1043,23 @@ MdPanelRef.prototype.open = function() {
   return this._$q(function(resolve, reject) {
     var done = self._done(resolve, self);
     var show = self._simpleBind(self.show, self);
+    var setOpened = function(response) {
+      self.isOpened = true;
+      return response;
+    };
+    var checkGroupMaxOpen = function(response) {
+      if (self.config.groupName) {
+        self._$mdPanel._checkGroupMaxOpen(self.config.groupName);
+        return response;
+      } else {
+        return response;
+      }
+    };
 
     self.attach()
         .then(show)
+        .then(setOpened)
```

Comment:
@bradrich It seemed like the ordering was wrong, but I guess since you just close the original panel, it's not as important.